packaging/linux/sunshine.appdata.xml

```diff
@@ -11,6 +11,8 @@
             @PROJECT_LONG_DESCRIPTION@
         </p>
     </description>
+    <launchable type="desktop-id">sunshine.desktop</launchable>
```

Comment:
How will this work with the multiple desktop files?

We provide 2 for normal cases. 2 for Flatpak (X11 versus KMS).